cargo-concordium/src/build.rs

```diff
@@ -49,17 +49,15 @@ const TARGET: &str = "wasm32-unknown-unknown";
 /// Get the crate's metadata either by looking for the `Cargo.toml` file at the
 /// `--manifest-path` or at the ancestors of the current directory.
 ///
-/// If successful, the return value is a pair of metadata and all of the
-/// `cargo_args` except the `--manifest-path` and the path to the manifest file.
-/// This last part is used for reproducible builds. There we want to keep the
-/// remaining `cargo` arguments, but the manifest path does not make sense since
-/// the project is built from a specific location inside the container.
-pub(crate) fn get_crate_metadata(
```

Comment:
Found it confusing that this returned two things, especially when the latter return value is only used in one place. It's been moved to where it's needed (`build_in_container`)